src/main/resources/changelog/scripts/v-1.0.0/resource_graph/tables/resources.sql

```diff
@@ -9,9 +9,9 @@ CREATE TABLE if not exists resources (
   created_event_id bigint null,
   updated_event_id bigint null,
   active_event_id bigint null,
-  inventory_id uuid,
-  srs_id uuid,
-  index_date timestamp
+  instance_metadata_id integer null,
```

Comment:
I would reverse this relationship. The primary key for the instance_metadata should be the resource_hash for the resource in question, obviating the need for a serial id and would remove the need to change this table.

Reply:
That make sense. Done.